zebra-chain/src/block/tests/vectors.rs

```diff
@@ -202,7 +202,7 @@ fn block_test_vectors_height_mainnet() {
 fn block_test_vectors_height_testnet() {
     let _init_guard = zebra_test::init();
 
-    block_test_vectors_height(Testnet);
+    block_test_vectors_height(Testnet(None.into()));
```

Comment:
For ease of implementation, and minimising changes, this could be replaced with importing a `Testnet` constant at the top of the file.

(And the appropriate lint exceptions to allow the constant to have non-standard capitalisation.)

Reply:
Ah actually I don't think this is a good idea, because then every reviewer has to notice the difference between `Testnet` (variant) and `Testnet` (default public testnet constant).

Instead, we could have a `PUBLIC_TESTNET` constant, and do a mass replace?